tests/test_c_messaging.py

```diff
@@ -49,13 +49,22 @@ def test_delivery_accepted():
 
 
 def test_delivery_rejected():
-    rej_val = c_uamqp.Messaging.delivery_rejected(b'Failed', b'Test failure')
```

Comment:
We should also keep the original test - these extra fields should be optional, and default to None.

Reply:
added back, thanks for reminding